content/en/sensitive_data_scanner/_index.md

```diff
@@ -161,11 +277,24 @@ To redact the attribute:
 7. Optionally, add tags.
 8. Click **Add Rules**.
 
+[1]: https://app.datadoghq.com/organization-settings/sensitive-data-scanner/configuration
+[2]: /logs/log_configuration/processors/?tab=ui#remapper
+[3]: https://app.datadoghq.com/logs/pipelines
+{{% /tab %}}
+{{% tab "Using the Agent" %}}
+This functionality is not available for Sensitive Data Scanner using the Agent.
+{{% /tab %}}
+{{< /tabs >}}
+
 ## Out-of-the-box dashboard
 
-When Sensitive Data Scanner is enabled, an [out-of-the-box dashboard][13] summarizing sensitive data findings is automatically installed in your account. To access this dashboard, go to **Dashboards > Dashboards List** and search for `Sensitive Data Scanner Overview`.
+When Sensitive Data Scanner is enabled, an [out-of-the-box dashboard][13] summarizing sensitive data findings is automatically installed in your account. To access this dashboard, go to **Dashboards > Dashboards List** and search for "Sensitive Data Scanner Overview".
```

Comment:
For another PR, this section should be changed to a link to `Investigate Sensitive Data Issues`. The dashboard should be part of this doc. We should heavily recommend users to use the Summary page and heads-up the dashboard only for private beta users of SDS in Agent.

Reply:
cc @PronoyC  if you can take a look to this part.

Reply:
I'm confused by this comment. Is there a requested change on my part?

Reply:
Not on your part necessarily and definitely not in this PR. I've open a comment so we (Pronoy or I) can follow-up on this later.

Reply:
Acknowledged, thanks for clarifying

Reply:
yeah I can resolve this comment -- it's something to update in a future PR instead